PusherServer/PusherOptions.cs

```diff
@@ -39,6 +42,25 @@ public bool Encrypted
         }
 
         /// <summary>
+        /// The host of the HTTP API endpoint excluding the scheme e.g. api.pusherapp.com
+        /// </summary>
+        /// <remarks>
+        /// If the scheme is included when setting the value it will be stripped from the set value.
+        /// e.g. setting with a value of "https://api.pusherapp.com" will result in a value of "api.pusherapp.com".
+        /// </remarks>
+        public string Host
+        {
+            get
+            {
+                return _host;
+            }
+            set
+            {
+                _host = Regex.Replace(value, "^https?://", string.Empty);
```

Comment:
This seems like the wrong place to do that. If it's possible to pass in a URI then the cleanup could be done on the URI-accepting constructor for example.